pkg/scheduler/scheduler.go

```diff
@@ -901,3 +926,11 @@ func (s *Scheduler) recordScheduleResultEventForClusterResourceBinding(crb *work
 		s.eventRecorder.Event(ref, corev1.EventTypeWarning, events.EventReasonScheduleBindingFailed, schedulerErr.Error())
 	}
 }
+
+// IsUnschedulableBinding checks if the ResourceBinding is unschedulable.
+func IsUnschedulableBinding(bindingStatus workv1alpha2.ResourceBindingStatus) bool {
```

Comment:
How about moving this function to the util package?

Reply:
Done.